app/uk/gov/hmrc/eoricommoncomponent/frontend/domain/subscriptionStatusModels.scala

```diff
@@ -22,7 +22,13 @@ import uk.gov.hmrc.eoricommoncomponent.frontend.domain.messaging.CommonHeader
 
 import java.time.format.DateTimeFormatter
 
-case class SubscriptionStatusQueryParams(receiptDate: LocalDateTime, regime: String, idType: String, id: String) {
+case class SubscriptionStatusQueryParams(
+  receiptDate: LocalDateTime,
+  regime: String,
+  idType: String,
+  id: String,
+  originatingService: String
```

Comment:
We don't add this as a query param for subscription display, why are we doing it for subscription status?

Reply:
Fixed the code